README.md

```diff
@@ -71,7 +71,43 @@ Required input:
 </p>    
 </div>
 
-### Resources
+## <div align="center">High level features and techniques</div>
+
+MCT offers a range of powerful features to optimize models for efficient edge deployment. These supported features include:
+
+### Quantization Core Features
+
+🏆 **Mixed-precision search** [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mixed_precision_ptq.ipynb). Assigning optimal quantization bit-width per layer (for weights/activations) 
+
+📈 **Graph optimizations**. Transforming the model to be best fitted for quantization process.
+
+🔎 **Quantization parameter search**. Minimizing expected quantization-noise during thresholds search using methods such as MSE, No-Clipping and MAE.
```

Comment:
consider adding a link for tutorial here as well (we have threshold search tutorial, currently only in keras)
https://github.com/sony/model_optimization/blob/2f4f02456dc155b6c87bdfe2e70033b1bf75391a/tutorials/notebooks/mct_features_notebooks/keras/example_keras_activation_threshold_search.ipynb

Reply:
Great! done